jenkins/release-workflows/release-tag.jenkinsfile

```diff
@@ -30,35 +30,43 @@ pipeline {
             description: 'Version of the release.',
             trim: true
         )
-        choice(
-            choices: ['opensearch', 'opensearch-dashboards'],
-            name: 'PRODUCT',
-            description: 'Product for tag creation.'
-        )
     }
     environment {
-        DISTRIBUTION_MANIFEST = "$PRODUCT-$VERSION/manifest.yml"
+        OS_DISTRIBUTION_MANIFEST = "opensearch-$VERSION/manifest.yml"
+        OSD_DISTRIBUTION_MANIFEST = "opensearch-dashboards-$VERSION/manifest.yml"
     }
     stages {
-        stage('Create Release Tag') {
+        stage('Create Release Tag for OpenSearch components') {
             steps {
                 script {
-                    release_url = "https://artifacts.opensearch.org/releases/bundle/$PRODUCT/$VERSION/$PRODUCT-$VERSION-linux-x64.tar.gz"
-                    sh "curl -SLO $release_url && tar -xf $PRODUCT-$VERSION-linux-x64.tar.gz $PRODUCT-$VERSION/manifest.yml"
-                    echo "The release distribution manifest is downloaded to $DISTRIBUTION_MANIFEST"
+                    os_artifact_url = "https://artifacts.opensearch.org/releases/bundle/opensearch/$VERSION/opensearch-$VERSION-linux-x64.tar.gz"
+                    sh "curl -SLO $os_artifact_url && tar -xf opensearch-$VERSION-linux-x64.tar.gz opensearch-$VERSION/manifest.yml"
```

Comment:
Any reason to not use the env var here instead of `opensearch-$VERSION/manifest.yml`?

Reply:
Made the changes. Thanks